README.md

```diff
@@ -1,99 +1,224 @@
-
-Neovim client library and GUI, in Qt5.
-
 [![Build Status](https://circleci.com/gh/equalsraf/neovim-qt.svg?style=svg)](https://circleci.com/gh/equalsraf/neovim-qt)
 [![Build status](https://ci.appveyor.com/api/projects/status/c252f54mfjcuud8x/branch/master?svg=true)](https://ci.appveyor.com/project/equalsraf/neovim-qt/branch/master)
 [![Build Status](https://travis-ci.org/equalsraf/neovim-qt.svg?branch=master)](https://travis-ci.org/equalsraf/neovim-qt)
 [![codecov Status](https://codecov.io/gh/equalsraf/neovim-qt/branch/master/graph/badge.svg)](https://codecov.io/gh/equalsraf/neovim-qt)
 [![Build Status](https://dev.azure.com/equalsraf/neovim-qt/_apis/build/status/equalsraf.neovim-qt?branchName=master)](https://dev.azure.com/equalsraf/neovim-qt/_build/latest?definitionId=1&branchName=master)
 [![Downloads](https://img.shields.io/github/downloads/equalsraf/neovim-qt/total.svg?maxAge=2592000)](https://github.com/equalsraf/neovim-qt/releases)
 
-## Requirements
+# Neovim Qt
+
+Neovim Qt is a cross-platform Neovim GUI written in C++ and Qt. It provides a terminal-like experience.
+
+![NeovimQt Screenshot](https://user-images.githubusercontent.com/11207308/108295028-f79f1b80-7164-11eb-8420-e9950fa97cd0.png)
+
+
+## Installing Neovim Qt
+
+Neovim Qt is available on all platforms supported by Qt. Instructions for common platforms listed below.
+
+### Windows
+A Neovim Qt is bundled with all releases of Neovim on Windows.
+
+See the Neovim release page:
+
+ - [Nighly Release](https://github.com/neovim/neovim/releases/nightly)
+ - [Stable Release](https://github.com/neovim/neovim/releases/latest)
+
+Noevim is available as a [Chocolatey Package](https://chocolatey.org/packages/neovim/).
+
+Latest Stable:
+```
+choco install neovim
+```
+
+Pre-Release:
+```
+choco install neovim --pre
+```
+
+### MacOS
+
+You can use HomeBrew to install Neovim Qt: https://github.com/equalsraf/homebrew-neovim-qt.
+
+To install the latest release:
+```
+$ brew tap equalsraf/neovim-qt
+$ brew install neovim-qt
+```
+
+To install the latest development version:
+```
+$ brew tap equalsraf/neovim-qt
+$ brew install neovim-qt
+```
+
+### Linux
+
+Neovim Qt is packaged for many distributions, use your favorite package manager.
+
+If your distribution is not listed here, please create an Issue with instructions.
+
+#### ArchLinux
+
+NeovimQt is available from this [AUR Package](https://archlinux.org/packages/community/x86_64/neovim-qt/).
```

Comment:
This is not AUR package.

Reply:
Sorry, I don't run Arch...

Do you have better instructions/links you can provide? Thanks!

Reply:
`NeovimQt is available from this [AUR Package]` -> `NeovimQt is available in [community repository]`